### ISSUE TITLE
Suggestion: slip:beforeswipe event based on direction (left/right)

### ISSUE THREAD

**Issue by mauron85** [pornel/slip#42]
I would like to prevent swiping based on swipe direction (left/right). Not sure if this can be implemented.
Example:

```javascript
list.addEventListener('slip:beforeswipe', function(e, direction) {
	if (direction === 'left') {
		e.preventDefault();
	}
});
```

**Comment by Voyga** [comment]
+1 I'm also looking for this.

**Comment by darrynten** [comment]
I've solved this issue in #56 (amongst other enhancements and features)

You'll get the direction in `e.details.directionX` and `e.details.directionY`

**Comment by mauron85** [comment]
Thank you. Looks I like a big improvement. Will try it out.
